src/CommandingOfficers/AW3/BM/Olaf.java

```diff
@@ -38,10 +38,12 @@ public instantiator()
           + "Winter weather poses no problem for Olaf or his troops. Snow causes his firepower to rise, and his troops can move with no penalty.\n"
           + "(Normal movement and +20 attack in cold weather.)"));
       infoPages.add(new InfoPage(new Blizzard(null, null),
-            "Causes chill (doubled fuel consumption) for two days.\n"
+            "Causes cold weather (doubled fuel consumption) for two days.\n"
+          + "Disables temporary fog, if it's active.\n"
           + "+10 attack and defense.\n"));
       infoPages.add(new InfoPage(new WinterFury(null, null),
-            "A mighty blizzard causes two HP of damage to all enemy troops. The chill also persists for two days.\n"
+            "A mighty blizzard causes two HP of damage to all enemy troops. The cold weather also persists for two days.\n"
+          + "Disables temporary fog, if it's active.\n"
```

Comment:
Maybe "weather-induced" fog, instead of "temporary"? Are there any other sources of temporary fog? If someone sets the game weather to persistent rain (or smoke I guess), would this disable the fog of war then?

Reply:
"Weather-induced" would imply that the fog is tied to the weather state, which it isn't.
> Are there any other sources of temporary fog?

Not yet. Maybe eventually.
> If someone sets the game weather to persistent rain (or smoke I guess), would this disable the fog of war then?

Those weathers do not inherently make the game fog in this engine, and I don't really want to change that (since weather is local).
I guess it'd be doable to have fog rules apply anywhere a fog weather is in effect (but not otherwise), but I at least don't want to add that scope right now.

But to answer the question, "no".
If fog is on, it would counteract the vision penalty effects of those weathers.